docs/configuration/configuration.md

```diff
@@ -2889,6 +2889,7 @@ metadata_config:
 
 # HTTP client settings, including authentication methods (such as basic auth and
 # authorization), proxy configurations, TLS options, custom HTTP headers, etc.
+# enable_http2 defaults to false for remote-write.
```

Comment:
maybe for better visibility we can have this comment at the "default" `http_config.enable_http2`?

https://github.com/prometheus/prometheus/pull/15119/files assumed that section is a real common one.